### PR TITLE
Translation of contributing/contributor-swag

### PR DESCRIPTION
**Pull request by SamuelAlev**

### CODE REVIEW

File: docs/contributing/contributor-swag.md [hunk 1]
```diff
@@ -1,26 +1,26 @@
 ---
-title: Free Swag for Gatsby Contributors!
+title: Cadeaux gratuit pour les contributeurs de Gatsby !
 ---
 
-Gatsby relies on contributors from the open source community, and we want to make sure you know how much it means to us that you’re willing to put time and effort into [contributing to Gatsby](/contributing/how-to-contribute/).
+Gatsby s'appuie sur des contributeurs de la communauté open source, et nous voulons nous assurer que vous savez à quel point il est important pour nous que vous soyez prêt à consacrer du temps et des efforts dans la [contribution à Gatsby](/contributing/how-to-contribute/).
```
Comment (mindsers) [inline docs/contributing/contributor-swag.md]:
```suggestion
Gatsby s'appuie sur des contributeurs de la communauté open source, et nous voulons nous assurer que vous savez à quel point il est important pour nous que vous ayez envie de consacrer du temps et des efforts dans la [contribution à Gatsby](/contributing/how-to-contribute/).
```

File: docs/contributing/contributor-swag.md [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
+Pour montrer notre gratitude, _**tous ceux qui contribuent à Gatsby peuvent recevoir un cadeau gratuit !**_
 
-![Gatsby contributors wearing swag.](./images/gatsby-swag.jpg)
+![Les contributeurs de Gatsby portant le cadeau.](./images/gatsby-swag.jpg)
 
-### How to claim your free swag
+### Comment obtenir votre cadeau gratuit
 
-If you contribute to the Gatsby organization on GitHub, you can claim your swag by [logging into the Gatsby Store](https://store.gatsbyjs.org/) and requesting a discount code. With five or more contributions, you can claim your _**Level 2**_ swag.
+Si vous contribuez à l'organisation Gatsby sur GitHub, vous pouvez réclamer votre cadeau en vous [connectant dans la boutique Gatsby](https://store.gatsbyjs.org/) et en demandant un code de réduction. Avec cinq contributions ou plus, vous pouvez obtenir votre cadeau _**Level 2**_.
```
Comment (mindsers) [inline docs/contributing/contributor-swag.md]:
```suggestion
Si vous contribuez à l'organisation Gatsby sur GitHub, vous pouvez réclamer votre cadeau en vous [connectant à la boutique Gatsby](https://store.gatsbyjs.org/) et en demandant un code de réduction. Avec cinq contributions ou plus, vous pouvez obtenir votre cadeau de _**Niveau 2**_.
```

**Review by mindsers — CHANGES_REQUESTED** [review]
Great job! I propose some modifications, we can discuss it if needed. 😉

File: docs/contributing/contributor-swag.md (hunk 1 above)
Comment (SamuelAlev) [reply docs/contributing/contributor-swag.md]: Petit belgicisme de ma part ^^

Comment (alexislepresle) [reply docs/contributing/contributor-swag.md]: Autrement c'est bon  👍

**Review by mindsers — APPROVED** [review]

**Pull request merged**